brackend/tasks/tasks.py

```diff
@@ -89,11 +65,28 @@ def get_tournament_ids():
         return [u.id for u in users]
 
 
-def get_tournament_by_id(id):
+def get_tournament_by_id(t_id):
```

Comment:
What's wrong with using `id` as the param? We know it's a tournament Id from the name of the method

Reply:
`id` is a python builtin, it's considered bad style to override it

Reply:
RIP I did not know that - will avoid it in the future. Although TBH that's pretty dumb by python because I feel like ID could be used everywhere

Reply:
Yeah, I kinda agree, also the usecase of the `id` built in is pretty niche